lib/internal/main/test_runner.js

```diff
@@ -41,12 +42,13 @@ if (shardOption) {
     );
   }
 
-  const { 0: indexStr, 1: totalStr } = shardOption.split('/');
+  const { 0: indexStr, 1: totalStr } = StringPrototypeSplit(shardOption, '/');
 
-  const index = NumberParseInt(indexStr);
-  const total = NumberParseInt(totalStr);
+  const index = NumberParseInt(indexStr, 10);
+  const total = NumberParseInt(totalStr, 10);
```

Comment:
fwiw if it's NumberParseInt, the , 10 isn't required - only on the global one.

Reply:
on the **global** one it's **not required** as far as I know

Reply:
It's required in the conceptual sense, because `parseInt` will try to _guess_ the radix if you omit it.